### DISCUSSION TITLE
Connection to Server

### DISCUSSION THREAD

**Discussion by lutzb91** [astubenbord/paperless-mobile#64] [Q&A] [answered]
Hi,
I would Like to use this App, but how do I connect top the server? I entered the IP of my installation, but I'm unable top click the button in the bottom of the App, as it ist greyed out.

**Comment by astubenbord** [comment, marked as answer]
Hi, what format does your address have? Did you provide a Scheme (i. e. http/https)?

**Reply by lutzb91** [reply]
Yes, i entered "http://192.168.178.124:8000" as address.

**Reply by astubenbord** [reply]
Is there a message below? Probably not right?
The button not changing to a tappable state indicates that a connection to the server could not be established. Based on the IP I assume it is a local one, maybe try disconnecting from mobile data?

**Reply by lutzb91** [reply]
I tried disconnecting from mobile data but the button stays in that state.  I did not specify a client certificate, as i don't have one and actual don't use TLS authentification. Is it correct, that the app only has a input field for the server address?

**Reply by astubenbord** [reply]
Yes, as I first want to make sure that a paperless instance is actually running on the specified server and port. There should be a client certificate Form field as well.

**Reply by lutzb91** [reply]
Yes, there are the two fields for the address and the client certificate. Do I have to fill in the certificate or can i leave this field empty?

**Reply by astubenbord** [reply]
But I guess I see what you mean, error handling is not that sophisticated for login. I'll create an issue and see what I can do in the next version.

**Reply by astubenbord** [reply]
> Yes, there are the two fields for the address and the client certificate. Do I have to fill in the certificate or can i leave this field empty?

You can leave it empty if not required.

**Reply by lutzb91** [reply]
Got the update but seems as if the app seems happy with the entered address. Button stays unclickable.

![Screenshot_2023-01-12-18-59-33-478_de astubenbord paperless_mobile](https://user-images.githubusercontent.com/19185136/212144020-cb2879d4-a460-4848-9c95-69dfcb2e4c96.jpg)

**Reply by lutzb91** [reply]
In Chrome everything is reachable.

![Screenshot_2023-01-12-19-03-35-313_com android chrome](https://user-images.githubusercontent.com/19185136/212144431-42ab45bd-23ac-4e2f-b693-9820c82eab80.jpg)

**Reply by astubenbord** [reply]
Yeah, that's weird. How do you enter the address? Do you press the enter button on your keyboard after you entered it? I think if you don't do it currently the system will not update the reachability status.

**Reply by chetwisniewski** [reply]
On my phone it is impossible to get past this prompt. There is no enter key on the on-screen keyboard and I even plugged in a real keyboard and hit the enter key but still doesn't work.

**Comment by lutzb91** [comment]
Ah, that solved it. Thanks!

Anton Stubenbord ***@***.***> schrieb am Do., 12. Jan. 2023,
23:03:
